From 3180da37ae457f84b076db71dd0f13b41d0a195e Mon Sep 17 00:00:00 2001
From: Athenas Jimenez Gonzalez <athenas.jimenez.gonzalez@intel.com>
Date: Wed, 4 May 2016 21:48:11 +0000
Subject: [PATCH] build

---
 tools/cpp/CROSSTOOL | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/cpp/CROSSTOOL b/tools/cpp/CROSSTOOL
index a668263..aa3e152 100644
--- a/tools/cpp/CROSSTOOL
+++ b/tools/cpp/CROSSTOOL
@@ -98,6 +98,7 @@ toolchain {
   # absolute locations and has no remote execution, yet. However, this will need
   # to be fixed, maybe with auto-detection?
   cxx_builtin_include_directory: "/usr/lib/gcc/"
+  cxx_builtin_include_directory: "/usr/lib64/gcc/x86_64-generic-linux/8.3.1/"
   cxx_builtin_include_directory: "/usr/local/include"
   cxx_builtin_include_directory: "/usr/include"
   tool_path { name: "gcov" path: "/usr/bin/gcov" }
@@ -131,6 +132,7 @@ toolchain {
   compiler_flag: "-U_FORTIFY_SOURCE"
   compiler_flag: "-D_FORTIFY_SOURCE=1"
   compiler_flag: "-fstack-protector"
+  compiler_flag: "-O2"
   linker_flag: "-Wl,-z,relro,-z,now"
 
   # Enable coloring even if there's no attached terminal. Bazel removes the
-- 
2.8.1

