--- bazel-0.1.2/tools/cpp/CROSSTOOL~	2015-11-20 06:29:05.000000000 -0500
+++ bazel-0.1.2/tools/cpp/CROSSTOOL	2015-12-29 17:36:17.842484208 -0500
@@ -92,6 +92,8 @@
   # used by gcc. That works because bazel currently doesn't track files at
   # absolute locations and has no remote execution, yet. However, this will need
   # to be fixed, maybe with auto-detection?
+  cxx_builtin_include_directory: "/usr/lib64/gcc/x86_64-generic-linux/5.2.0/include/"
+  cxx_builtin_include_directory:  "/usr/lib64/gcc/x86_64-generic-linux/5.2.0/include-fixed/"
   cxx_builtin_include_directory: "/usr/lib/gcc/"
   cxx_builtin_include_directory: "/usr/local/include"
   cxx_builtin_include_directory: "/usr/include"
--- bazel-0.1.2/tools/cpp/CROSSTOOL~	2015-12-29 17:36:17.000000000 -0500
+++ bazel-0.1.2/tools/cpp/CROSSTOOL	2015-12-29 17:45:59.931207097 -0500
@@ -127,6 +127,7 @@
   compiler_flag: "-U_FORTIFY_SOURCE"
   compiler_flag: "-D_FORTIFY_SOURCE=1"
   compiler_flag: "-fstack-protector"
+  compiler_flag: "-O2"
   linker_flag: "-Wl,-z,relro,-z,now"
 
   # Enable coloring even if there's no attached terminal. Bazel removes the
